<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../3 SEMESTRE/ED 1 (3 SEMESTRE)/Downloads/WhatsApp Image 2023-09-23 at 17.26.09.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>671</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manipulação de dados Exercicio 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMostrarDados">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>191</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>MOSTRAR DADOS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../3 SEMESTRE/ED 1 (3 SEMESTRE)/Downloads/inserir.png</normaloff>../../../../3 SEMESTRE/ED 1 (3 SEMESTRE)/Downloads/inserir.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EncontrarBS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Encontrar na Grid</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ResulBS">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dados dos professores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Forma de Ordenação: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Matricula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Departamento-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Titulacao-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tipo de contrato-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Departamento-Titulacao-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Departamento-Tipo de contrato-Nome</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonBuscarNomeArquivo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>BUSCAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../3 SEMESTRE/ED 1 (3 SEMESTRE)/Downloads/pesquisa.png</normaloff>../../../../3 SEMESTRE/ED 1 (3 SEMESTRE)/Downloads/pesquisa.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_BS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Busca Sequencial-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Sequencial-Matricula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Binaria-Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Binaria-Matricula</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeDoArquivo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do arquivo:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>741</width>
      <height>541</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Matricula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dep</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titulacao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
